### ISSUE TITLE
DEFAULT value of DATETIME is in wrong format when ALTER TABLE

### ISSUE THREAD

**Issue by xlight** [Sequel-Ace/Sequel-Ace#1566]
- Sequel Ace Version (including build number): Version 3.5.3 Build 20035
- Sequel Ace Source (App Store/GitHub/Homebrew): GitHub
- macOS Localization: Chinese

**Description**
ERROR when I ALTER  a Field with Type DATETIME .
with Wrong SQL:
```
ALTER TABLE `ks_samples_2` CHANGE `producPickTime` `producPickTime` DATETIME
NULL
DEFAULT 0000-00-00 00:00:00
```

**Expected Behaviour**
the SQL should be:
```
ALTER TABLE `ks_samples_2` CHANGE `producPickTime` `producPickTime` DATETIME
NULL
DEFAULT '0000-00-00 00:00:00'
```

<img width="260" alt="image" src="https://user-images.githubusercontent.com/77911/192671155-9695b5fb-20b2-486a-aca4-8b1a622f6cf3.png">

**Comment by xlight** [comment]
this issue  still exists on “Version 4.0.1 Build 20039”

**Comment by Jason-Morcos** [comment]
Please try the latest beta and confirm it's all good - the fix for this issue is included in the latest 4.0.5+ beta!

https://github.com/Sequel-Ace/Sequel-Ace/releases

**Comment by xlight** [comment]
> Please try the latest beta and confirm it's all good - the fix for this issue is included in the latest 4.0.5+ beta!
> 
> https://github.com/Sequel-Ace/Sequel-Ace/releases

confirmed ok
